### PR TITLE
New quickstart - demonstrating remote task execution with streams.

### PR DESCRIPTION
**Pull request by andyuk1986**

### CODE REVIEW

File: remote-tasks-with-streams/README.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+
+       For Linux:    mkdir $JDG_HOME/modules/system/layers/base/org/jboss/as/quickstarts/remote-task-with-streams/main
+       For Windows:  MD %JDG_HOME%\modules\system\layers\base\org\jboss\as\quickstarts\remote-task-with-streams\main
+         
+       For Linux:    cp target/remote-tasks-with-streams-lib.jar $JDG_HOME/modules/system/layers/base/org/jboss/as/quickstarts/remote-task-with-streams/main/
+       For Windows:  COPY "target\remote-tasks-with-streams-lib.jar" "%JDG_HOME%\modules\system\layers\base\org\jboss\as\quickstarts\remote-task-with-streams\main\"
+
+* Copy the `resources/module.xml` file to `$JDG_HOME/modules/system/layers/base/org/jboss/as/quickstarts/remote-task-with-streams/main/`:
+
```
Comment (slaskawi) [inline remote-tasks-with-streams/README.md:93]: An empty line to be removed here.

**Review by slaskawi — APPROVED** [review]
Looks very nice! I really like this QuickStart.

@mgencur @VaclavDedik Any other comments? If not, I'll integrate it.

**Comment by VaclavDedik** [comment]
Looks good, can you just add the quickstart to the source assembly please? It won't be packaged for JDG 7.1 if it's not there:

https://github.com/jboss-developer/jboss-jdg-quickstarts/blob/jdg-7.1.x/src/main/assemblies/sources.xml

**Comment by mgencur** [comment]
Looks good to me ;-)

**Comment by andyuk1986** [comment]
Thanks everyone for the review. 
@slaskawi I have removed the extra line on the line that you have mentioned and also fixed a bit the formatting, now the README.md looks good with github viewer. 

@VaclavDedik I have added the quickstart to the source assembly as you have requested.

**Comment by VaclavDedik** [comment]
Integrated, thanks!

**Pull request closed without merging**